0_explore_data/cluster_by_accessory_gene.ipynb

```diff
@@ -2174,33 +2174,33 @@
        "</div>"
       ],
       "text/plain": [
-       "                PA2004      PA2174      PA1173      PA0101      PA1242  \\\n",
-       "ERX1272616   87.578274   55.796995   23.440490   63.993977   34.513605   \n",
-       "ERX1272618  223.430398  107.107103   91.663753  292.676385  256.309788   \n",
-       "ERX1272619  240.507849  114.761277   93.725605  248.220929  262.010980   \n",
-       "ERX1296067  433.829656  549.973029  316.661489  100.429857  177.972639   \n",
-       "ERX1296068  529.491131  538.185333  284.343151  141.102616  257.832963   \n",
+       "                 PA4655      PA1271     PA1705      PA3140      PA2312  \\\n",
```

Comment:
why did these genes change?

Reply:
I just checked that these column values are unchanged but the ordering of the columns are different, probably due to some randomness in the set operations I used to select gene ids/columns. This is the same explanation for the columns below